specs/pipelines/http-resolvers.spec

```diff
@@ -0,0 +1,22 @@
+PIPELINES-31
+# http resolvers spec
+
+Pre condition:
+  * Validate Operator should be installed
```

Comment:
include an empty line after preconditions to increase readability

Reply:
Done